string name
---
bool success
geometry_msgs/Point position
string message
